frontend/src/components/GddSend/TransactionResultLink.vue

```diff
@@ -17,17 +25,24 @@
 </template>
 <script>
 import ClipboardCopy from '../ClipboardCopy.vue'
+import FigureQrCode from '../QrCode/FigureQrCode.vue'
 
 export default {
   name: 'TransactionResultLink',
   components: {
     ClipboardCopy,
+    FigureQrCode,
   },
   props: {
     code: {
       type: String,
       required: true,
     },
   },
+  computed: {
+    link() {
+      return `${window.location.origin}/redeem/${this.code}`
+    },
+  },
```

Comment:
this is duplicate code from `clipboard-copy` - remove code from clipbaord-copy and have a link prop. Also note the conflict with the PR of @Tirokk

Reply:
code is removed from clipbaord-copy